capstone-frontend/src/components/SearchUserModal.jsx

```diff
@@ -45,6 +45,10 @@ export class SearchUserModal extends Component {
       else if (this.props.type === 'booklists' && this.props.bookList) {
         searchResults = searchResults.filter(searchItem => searchItem.id !== window.localStorage.getItem('userID'))
       }
+      // Owner cannot be a Member
+      else if (this.props.type === 'clubs' && this.props.club) {
+        searchResults = searchResults.filter(searchItem => searchItem.id !== window.localStorage.getItem('userID'))
+      }
```

Comment:
Is there any way to combine this and the previous statement? Maybe with a `||`? The filter line is exactly the same it seems

Reply:
I was thinking of combing the conditionals initially, but I feel separating them makes the code more readable. Let me know if I should.